scripts/core/editor3/styles.scss

```diff
@@ -526,6 +526,10 @@ $editor-styleButton-size: 3rem;
 	[style^="color"] { // used for suggestions
 		background-color: rgba(0, 180, 0, 0.2);
 	}
+
+	[style^="color: rgb(255"] { // used for delete suggestions
```

Comment:
That's a bit dirty, is it hard to add a class name?

Reply:
it's added via highlights, afaik there is no way to add class for this

Reply:
And it totally depends on `SpaceDecorator`. Speaking of `SpaceDecorator`, you can edit its renderer - `SpaceComponent` and add classes for suggestions. Shouldn't be too hard :)